include/mgba-util/platform/posix/threading.h

```diff
@@ -94,6 +94,8 @@ static inline int ThreadSetName(const char* name) {
 #elif defined(__HAIKU__)
 	rename_thread(find_thread(NULL), name);
 	return 0;
+#elif defined(__NetBSD__)
+	return pthread_setname_np(pthread_self(), "%s", (void *)name);
```

Comment:
Why does this need to be cast to void*? This is C code.

Reply:
Technically unnecessary, but it gets rid of a "discards const" warning.

Reply:
Ohh fair enough. Please add a space between the cast and the variable then.